src/Parsers/ASTCreateQuery.cpp

```diff
@@ -32,6 +32,11 @@ ASTPtr ASTStorage::clone() const
     if (settings)
         res->set(res->settings, settings->clone());
 
+    if (comment)
+    {
+        res->comment = comment->clone();
+    }
```

Comment:
May be use `set` method as in the cases below?

Reply:
Ok.

Reply:
They differ only in type of pointer.